src/AutoBuild.jl

```diff
@@ -1268,80 +1275,98 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
         print(io, jll_jl)
     end
 
-    product_names(p::ExecutableProduct) = p.binnames
-    product_names(p::FileProduct) = p.paths
-    product_names(p::LibraryProduct) = p.libnames
-    product_names(p::FrameworkProduct) = p.libraryproduct.libnames
+    is_yggdrasil = get(ENV, "YGGDRASIL", "false") == "true"
+    ygg_head = is_yggdrasil ? LibGit2.head("..") : ""
+    src_name_first = uppercase(first(src_name))
+    print_source(io, s::ArchiveSource) = println(io, "* compressed archive: ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    print_source(io, s::GitSource) =     println(io, "* git repository: ", s.url, " (revision: `", s.hash,"`)")
+    print_source(io, s::FileSource) =    println(io, "* remote file: ", s.url, " (SHA256 checksum: `", s.hash,"`)")
+    function print_source(io, s::DirectorySource)
+        print(io, "* files in local directory: ")
+        if is_yggdrasil
+            println(io, "[`", s.path, "`](https://github.com/JuliaPackaging/Yggdrasil/tree/", ygg_head, "/", src_name_first, "/", src_name, "/", basename(s.path), ")")
+        else
+            println(io, "`", s.path, "`")
+        end
+    end
+    function print_jll(io, dep)
+        depname = getname(dep)
+        if is_yggdrasil
+            # In this case we can easily add a direct link to the repo
+            println(io, "* [`", depname, "`](https://github.com/JuliaBinaryWrappers/", depname, ".jl)")
+        else
+            println(io, "* `", depname, ")`")
+        end
+    end
+    print_product(io, p::Product) = println(io, "* `", typeof(p), "`: `", variable_name(p), "`")
     # Add a README.md
     open(joinpath(code_dir, "README.md"), "w") do io
         print(io,
               """
-              # $(src_name)_jll.jl
-
-              This is an autogenerated package constructed using [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).
-
-              For more details about JLL packages, see `BinaryBuilder.jl` [documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/).
-
-              """,
-              iszero(length(keys(products_info))) ? "" :
-              """
-              ## Products
-
-              The code bindings within this package are autogenerated from the following `Products` defined within the `build_tarballs.jl` file that generated this package:
-
-              ```julia
-              products = [
-              $(join(collect("    $(typeof(p))($(product_names(p)), :$(variable_name(p)))" for (p, _) in products_info), ",\n"))
-              ]
-              ```
-
-              """,
-              """
-              ## Usage example
-
-              For example purposes, we will assume that the following products were defined in the imaginary package `Example_jll`:
+              # `$(src_name)_jll.jl` (`v$(build_version)`)
```

Comment:
Why is the version in code quotes? The result looks a bit funny, IMHO. How about this as an alternative? (I realize this is a bit of bikeshedding; feel free to ignore)
```suggestion
              # `$(src_name)_jll.jl` $(build_version)
```

Reply:
Done